plugin/tables/bitlinks.py

```diff
@@ -60,15 +62,16 @@ def __init__(
 
     @property
     def resolver(self):
-        return BitlinksResolver(table=self)
+        return BitlinksResolver(table=self, link_filter=self.link_filter)
 
 
 class BitlinksResolver(TableResolver):
-    def __init__(self, table=None) -> None:
+    def __init__(self, table=None, link_filter = []) -> None:
         super().__init__(table=table)
+        self.link_filter = link_filter
 
     def resolve(self, client: Client, parent_resource) -> Generator[Any, None, None]:
-        for bitlink in client.client.list_bitlinks():
+        for bitlink in filter(lambda l: len(self.link_filter) == 0 or l["id"] in self.link_filter, client.client.list_bitlinks()):
```

Comment:
Just curious, shouldn't we add bit.ly prefix here or doc to only use the ID?

Reply:
the id is composed of the domain and the link id. While default is `bit.ly/<link>`, we use `cql.ink/<link>`, for example.